pgsync/sync.py

```diff
@@ -69,7 +69,7 @@ def __init__(
         )
         self.es = ElasticHelper()
         self.__name = re.sub(
-            "[^0-9a-zA-Z_]+", "", f"{self.database}_{self.index}"
+            "[^0-9a-zA-Z_]+", "", f"{self.database.lower()}_{self.index}"
```

Comment:
there is one more change that is required for this to work. This is the channel name. 
I need to do some regression and add some tests before merging this.

Reply:
It'd be nice to accept also the dash character in the database name